src/cmd.ts

```diff
@@ -366,6 +367,21 @@ async function cmdBuild(argv: BuildCmdArgvType) {
     return fs.write(rssPath, feed.xml({ indent: true }));
   };
 
+  const castPostsToFarcaster = (username: string, privateKey: string) => {
+    const farcaster = new FarcasterFeed(username, privateKey);
+    pages
+      .filter((page) => page.context.title && page.context.date)
+      .sort((a, b) => new Date(a.context.date).getTime() - new Date(b.context.date).getTime())
+      .forEach((page) => {
+        const post: Post = {
+          title: page.context.title,
+          url: `${appContext.base_url}${page.urlPath}`,
+        };
+        farcaster.addPost(post);
+      });
+    return farcaster.castPosts();
```

Comment:
Can you 'cast' multiple things in farcaster? If it's only posts that are 'casted', then I'd also remove the `Posts` from `castPosts`. One word makes for cleaner APIs assuming it doesn't add confusion / reduce clarity.

```typescript
farcaster.cast(posts)
```

Reply:
This is a nicer API :+1:

I am not sure that this is an official farcaster lib (it might be just a first pass from someone) so that upgrade is likely doable!

I will say, however, that API wise it's better to post a list, but product wise it's worse. It encourages posting multiple things in a row which is spammy and undesirable behavior in the twitter like product.